spec/models/reducers/consensus_reducer_spec.rb

```diff
@@ -18,18 +18,18 @@ def build_extracts(choices)
 
   describe '#process' do
     it 'processes when there are no classifications' do
-      expect(reducer.reduction_data_for([],nil)).to include({"num_votes" => 0})
+      expect(reducer.reduce_into([], create(:subject_reduction)).data).to include({"num_votes" => 0})
```

Comment:
Would a `build(:subject_reduction)` suffice, or do we really need it to be in the database?